### PR TITLE
Add AnimeSub.LT page

### PR DESCRIPTION
**Pull request by PcmKomanda**

### CODE REVIEW

File: src/pages/AnimeSub/main.ts [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+  },
+};
+
+function getId() {
+  const id = j.$('[rel="shortlink"]').attr('href')?.split('p=')[1];
+  if (id) return id;
+  return undefined;
+}
+
+function malId(provider) {
```
Comment (henrik9999) [inline src/pages/AnimeSub/main.ts:79]: this function isnt used

**Review by henrik9999 — CHANGES_REQUESTED** [review]

File: src/pages/AnimeSub/main.ts (hunk 1 above)
Comment (PcmKomanda) [reply src/pages/AnimeSub/main.ts]: Whoops. You're right. 😅

Comment (henrik9999) [reply src/pages/AnimeSub/main.ts]: now you are not using the anilist link

**Comment by henrik9999** [comment]
the site has anime and manga so you have to set the type in the init function accordingly `page.type = 'anime'` or `page.type = 'manga`

**Comment by PcmKomanda** [comment]
> page.type = 'anime' or page.type = 'manga'

Property 'type' does not exist on type 'SyncPage'

**Comment by lolamtisch** [comment]
Should be `AnimeSub.type` what can be set.
https://github.com/MALSync/MALSync/blob/master/src/pages/Proxer/main.ts#L119

**Pull request closed without merging**